src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final int maxFigure = 5;
+    private Random random;
+    private ColorSupplier colorSupplier;
+    
+    public FigureSupplier() {
+        colorSupplier = new ColorSupplier();
+        random = new Random();
+    }
+
+    public Figure getRandomFigure() {
+        int index = random.nextInt(maxFigure);
+        int num1 = random.nextInt(9) + 1;
```

Comment:
bad practice to use numbers for naming

create separate method for r`andom.nextInt(9) + 1` and use it for each type of figure

also, what is it 9?